src/headerdef.rs

```diff
@@ -10,6 +10,10 @@ pub enum HeaderDef {
     Cc,
     Disposition,
     OriginalMessageId,
+
+    /// Delta Chat extension for message IDs in combined MDNs
+    XAdditionalMessageIds,
```

Comment:
This breaks with the header naming style we otherwise do ("Chat-Version", "Autocrypt", "Chat-Group-Name" etc.), namely that we don't use the "X-" prefix.  I think it's better to stay internally consistent within DC so not use the "X-" prefix here -- unless there is a very good reason not to.

Reply:
Looks like this practice is even deprecated as stated in https://tools.ietf.org/html/rfc8098#appendix-A and https://tools.ietf.org/html/rfc6648

Ok, I am going to add a commit to change it.